app/main.py

```diff
@@ -1,8 +1,12 @@
 from fastapi import FastAPI
-from app.routes.gateway import router
+from app.routes.api_routes import router  # Updated import
```

Comment:
Lets remove gippity comment here :laughing: 

```suggestion
from app.routes.api_routes import router
```

Reply:
hehe